src/rust/ide/src/double_representation/definition.rs

```diff
@@ -166,16 +167,22 @@ impl DefinitionName {
         Some(DefinitionName {extended_target,name})
     }
 
-    /// Checks if the given definition name is a member of given type.
-    pub fn extends_type(&self, parent_name:&str, expected_typename:&str) -> bool {
+    /// Checks if the given definition name is a method defined on given expected atom name.
+    ///
+    /// E.g. `Main.foo` is a method of `Main`. Also, if `Main` is a name of module, than root-scoped
+    /// definition named `foo` will be treated as an extension method on `Main`. For such case the
+    /// `parent_name` is provided - it is a name of module or type where the definition is located.
```

Comment:
It's still a little bit unclear to me. _For such case_ suggests, that the parent_name is like optional argument. I would rather say "To handle such case properly, function takes 'parent_name' - ...".